exec.js

```diff
@@ -69,9 +77,10 @@ function checkNotis() {
 			throw error
 		}
 		for (noti of data) {
-			if (noti.status
-					&& noti.status.content
-					&& noti.status.content.includes('go now')) {
+			if ((ownAcct !== undefined || noti.account.acct !== ownAcct)
+			    && noti.status
```

Comment:
Why 4 spaces here? I'd rather not mix em. Can we get two tabs intsead?

Reply:
if you want alignment, you have to mix them
here's what the old version looks like with tabstop set to 8 (default for most browsers):
![image](https://user-images.githubusercontent.com/538336/47051724-0cb9a900-d16b-11e8-9f84-77933c453280.png)

Reply:
so i'd have either indent all the conditions, removing alignment
or keep the spaces, preserving alignment

Reply:
Yeah that looks good to me. I prefer to keep the conditional away from the code because of the KNR `{` style

Reply:
the *screenshot* looks good to you?

Reply:
Yeah.

Reply:
I don't really care though, do what you want haha

Reply:
Besides, I use 4 spaces in my editor